src/main/java/com/google/devtools/build/lib/exec/SpawnInputExpander.java

```diff
@@ -222,7 +223,8 @@ private static void addInputs(
       NestedSet<? extends ActionInput> inputFiles,
       ArtifactExpander artifactExpander,
       PathFragment baseDirectory) {
-    List<ActionInput> inputs = ActionInputHelper.expandArtifacts(inputFiles, artifactExpander);
+    List<ActionInput> inputs = ActionInputHelper.expandArtifacts(inputFiles, artifactExpander,
+        true);
```

Comment:
Please add an inline argument comment `/* keepEmptyTreeArtifacts= */` for boolean value. Same below.

Reply:
I went over the usages and added inline comments.